src/components/Login/Login.js

```diff
@@ -70,30 +79,30 @@ class Login extends Component {
     const isEnabled = this.validateForm();
 
     return(
-    <main className="login-container">
+    <section className="login-container">
       <form>
         <h1>Login</h1>
         <span className="error">{this.state.error}</span>
         <label htmlFor="email">Email</label>
         <input
           type="text"
           name="email"
-          placeholder="enter email"
+          placeholder="Enter email"
           onChange={this.handleUpdate}
```

Comment:
These name changes may break the Login.test, but it's ok. we can fix it in the next PR.